src/python-problem/tests/Makefile.am

```diff
@@ -4,11 +4,8 @@ check_SCRIPTS = $(TESTS)
 EXTRA_DIST = $(check_SCRIPTS) util.py base.py
 
 if BUILD_PYTHON3
-check-local:
-	echo "Running tests with python3"; \
-	exc=0; \
-	for test in $(TESTS); do \
-		python3 $$test || exc=1; \
-	done; \
-	exit $$exc
+if HAVE_PYTEST
+check-local: $(TESTS)
+	pytest-3 --cov=problem --capture=no --verbose $(TESTS)
```

Comment:
```
ERROR: usage: pytest-3 [options] [file_or_dir] [file_or_dir] [...]                                                     
pytest-3: error: unrecognized arguments: --cov=problem                                                                                                                                                                                        
  inifile: None                                                                                                        
  rootdir: /home/mkutlak/ABRT/abrt/build/abrt-2.12.2.67.gb050/src/python-problem/tests
```

Reply:
Ah, good catch, I think it’s a plugin. Better to not care about coverage here, then, but rather in Jenkins.